lib/endoscope/build.rs

```diff
@@ -0,0 +1,34 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+use anyhow::Result;
```

Comment:
NIT: Nothing in this file requires `anyhow::Result` so I don't think there's any reason to bring `anyhow` in as a dev dependency.